### PR TITLE
fix : insert lineprotocol error

### PR DESCRIPTION
**Pull request by Subsegment**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?

[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #.

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

File: coordinator/src/writer.rs [hunk 1]
```diff
@@ -395,17 +395,23 @@ impl PointWriter {
...
-            );
-
-            span_recorder.error(err.to_string());
 
-            return self
-                .write_to_handoff(vnode_id, node_id, tenant, precision, data)
-                .await;
+        if let Err(ref err) = result {
+            if matches!(*err, CoordinatorError::FailoverNode { .. })
+                || err.error_code().to_string() == *"020005"
```
Comment (roseboy-liu) [inline coordinator/src/writer.rs:401]: use TSKV::*  Error ？

File: tskv/src/schema/schemas.rs [hunk 2]
```diff
@@ -223,8 +234,19 @@ impl DBschemas {
...
-                                .ok_or(MetaError::Retry)?;
+                                .map_err(|e| MetaError::Retry { msg: e.to_string() })?;
+                            let schema_get = match schema_get {
+                                None => self
+                                    .client
+                                    .get_tskv_table_schema_by_meta(&schema.db, &schema.name)
+                                    .await
+                                    .map_err(|e| MetaError::Retry { msg: e.to_string() })?
+                                    .ok_or(MetaError::Retry {
+                                        msg: "Table not found when retry update table schema"
```
Comment (roseboy-liu) [inline tskv/src/schema/schemas.rs:245]: this code makes me confused， map_err()?.ok_or()?

File: tskv/src/schema/schemas.rs [hunk 3]
```diff
@@ -223,8 +234,19 @@ impl DBschemas {
...
+                                            .to_string(),
+                                    })?,
+                                Some(schema) => schema,
+                            };
                             let mut schema = schema.as_ref().clone();
                             schema.schema_id = schema_get.schema_id + 1;
```
Comment (roseboy-liu) [inline tskv/src/schema/schemas.rs:251]: schema_id should update in the meta server

**Review by roseboy-liu — APPROVED** [review]

File: coordinator/src/writer.rs (hunk 1 above)
Comment (Subsegment) [reply coordinator/src/writer.rs:401]: Errors are nested, and the error_code method can unnest them

File: tskv/src/schema/schemas.rs (hunk 2 above)
Comment (Subsegment) [reply tskv/src/schema/schemas.rs:245]: Result<Option<T>>

File: tskv/src/schema/schemas.rs (hunk 3 above)
Comment (Subsegment) [reply tskv/src/schema/schemas.rs:251]: Add 1 to the schema id before each retry

File: coordinator/src/writer.rs [hunk 4]
```diff
@@ -395,17 +395,23 @@ impl PointWriter {
...
+                || err.error_code().to_string() == *"030025"
+            {
+                debug!(
```
Comment (roseboy-liu) [inline coordinator/src/writer.rs:404]: error？

**Review by roseboy-liu — APPROVED** [review]

**Pull request merged**